apps/stats/src/views/Stats/Newsletters/components/NewslettersKPIs.tsx

```
@@ -191,69 +195,77 @@ const NewsletterKPIs: React.FC<{
+                                        data={avgsData}
+                                        margin={{
+                                            top: 20
+                                        }}
+                                        onClick={(e) => {
+                                            if (e.activePayload && e.activePayload![0].payload.post_id) {
+                                                navigate(`/posts/analytics/beta/${e.activePayload![0].payload.post_id}`, {crossApp: true});
+                                            }
+                                        }}
+                                        onMouseLeave={() => setIsHoveringClickable(false)}
+                                        onMouseMove={(e) => {
+                                            setIsHoveringClickable(!!(e.activePayload && e.activePayload[0].payload.post_id));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)